src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -390,6 +390,15 @@ public Task<Compilation> GetCompilationAsync(CancellationToken cancellationToken
         }
 
         /// <summary>
+        /// Determines if the compilation returned by <see cref="GetCompilationAsync"/> has all the references it's expected to have.
+        /// </summary>
+        // TODO: make this public
+        internal Task<bool> GetCompilationHasIncompleteReferencesAsync(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
HasIncomplete feels very weird to me.  it's like having an "IsNotXXX" method/property.  Perhaps it should be "HasCompleteReferences"?